sdk/containerregistry/container-registry/samples-dev/uploadImage.ts

```diff
@@ -31,14 +30,29 @@ async function main() {
     }
   );
 
-  const uploadResult = await client.uploadBlob(Readable.from(BLOB_CONTENT));
-
-  // Calling downloadBlob on the uploaded blob (identified by the digest) gives a readable stream containing the blob's content.
-  const downloadResult = await client.downloadBlob(uploadResult.digest);
-
-  // The downloaded content can be piped to another stream, e.g. one created using fs.createWriteStream to download to a file.
-  // In this example, we pipe the content to the standard output stream.
-  downloadResult.content.pipe(process.stdout);
+  const config = Buffer.from("Sample config");
+  const uploadConfigResult = await client.uploadBlob(Readable.from(config));
+
+  const layer = Buffer.from("Sample layer");
+  const uploadLayerResult = await client.uploadBlob(Readable.from(layer));
+
+  const manifest: OciManifest = {
+    schemaVersion: 2,
```

Comment:
In .NET, we are setting schemaVersion in the OciImageManifest constructor - we should decide if we want to do this across languages.  @lmolkova - any concerns about this?

Also - nit: should OciManifest be `OciImageManifest`?

Reply:
Yup, I meant to do that rename. I'll hold off on the schemaVersion change for now, but it seems reasonable to me that we would specify it by default. The OciImageManifest type itself needs a bit of work I think -- at the moment, we're just using the generated type verbatim, but there are other small issues (e.g. `config` is optional in the generated type, but it's [required in the spec](https://github.com/opencontainers/image-spec/blob/main/manifest.md)).